apis/nucleus/src/components/listbox/interactions/keyboard-navigation/keyboard-nav-rows.js

```diff
@@ -75,11 +75,10 @@ export default function getRowsKeyboardNavigation({
         }
 
         // Without shift key
-        if (inSelection) {
-          keyboard.focusSelection();
-        } else {
+        if (!keyboard.focusSelection()) {
           currentTarget.blur();
-          keyboard.blur();
+          keyboard.blur(true);
+          event.stopPropagation();
```

Comment:
is this needed?
(there is already a event.stopPropagation(); at the end of the function, after the switch)